polars/polars-lazy/polars-plan/src/dsl/function_expr/strings.rs

```diff
@@ -1,11 +1,16 @@
 use std::borrow::Cow;
 
+#[cfg(feature = "regex")]
```

Comment:
I think this should be under feature `timezones` and then the `timezones` feature can activate `regex`.